openmetadata-ui/src/main/resources/ui/src/components/Users/Users.component.tsx

```diff
@@ -101,6 +101,19 @@ const Users = ({ userData, queryFilters, updateUserDetails }: Props) => {
     setPreviewAsset(asset);
   }, []);
 
+  const handleTabRedirection = useCallback(() => {
+    if (!isLoggedInUser && activeTab === UserPageTabs.ACCESS_TOKEN) {
+      history.push({
+        pathname: getUserPath(decodedUsername, UserPageTabs.ACTIVITY),
+        search: location.search,
+      });
+    }
+  }, [activeTab, decodedUsername, isLoggedInUser]);
+
+  useEffect(() => {
+    handleTabRedirection();
+  }, [activeTab]);
```

Comment:
```suggestion
  }, []);
```

We will do this only for page load and after that it's not required